hw/regs/carfield_regs.csv

```diff
@@ -48,6 +48,7 @@ L2_CLK_DIV_VALUE,24,rw,hro,1,1,L2 Memory clk divider value
 HOST_FETCH_ENABLE,1,ro,hro,0,0,Host Domain fetch enable
 SAFETY_ISLAND_FETCH_ENABLE,1,rw,hro,0,0,Safety Island fetch enable
 SECURITY_ISLAND_FETCH_ENABLE,1,rw,hro,0,0,Security Island fetch enable
+PULP_CLUSTER_BOOT_ENABLE,1,rw,hro,0,0,PULP Cluster boot enable

```

Comment:
I would put it at the end of the memory map, so that the `fetch` signals are grouped together

Reply:
@yvantor :crab:

Reply:
Sure!

Reply:
> @yvantor 🦀

:golfing_man:

Reply:
Done